src/arraytraits.rs

```diff
@@ -207,10 +207,9 @@ unsafe impl<S, D> Send for ArrayBase<S, D>
     where S: Send + Data, D: Send
 { }
 
-
-#[cfg(feature = "rustc-serialize")]
+#[cfg(any(feature = "rustc-serialize", feature = "serde"))]
 // Use version number so we can add a packed format later.
-static ARRAY_FORMAT_VERSION: u8 = 1u8;
+pub static ARRAY_FORMAT_VERSION: u8 = 1u8;
```

Comment:
This should be a `const` now. I don't think const existed when the old code was first written.